### PR TITLE
Add specs to testnet participation guide

### LINKED ISSUE #4061: Hardware requirements aren't in testnet participation guide
#### Problem

Hardware requirements for testnet participation are in a Google doc whereas everything else about setup is in the book.

#### Proposed Solution

Consolidate.

tag: @sakridge, @ericlwilliams, @danpaul000

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Hardware specs not in testnet guide.

#### Summary of Changes

Add some specs.

Fixes #4061

### CODE REVIEW

File: book/src/testnet-participation.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ traversal issues.  A cloud-hosted machine works best.  **Ensure that IP ports
 Prebuilt binaries are available for Linux x86_64 (Ubuntu 18.04 recommended).
 MacOS or WSL users may build from source.
 
+For a performance testnet with many transactions we have some preliminary recomended setups:
+| | Low end | Medium end | High end |
+| --- | ---------|------------|----------|
+| CPU | AMD Threadripper 1900x | AMD Threadripper 2920x | AMD Threadripper 2950x |
+| RAM | 16GB | 32GB | 64GB |
+| OS Drive | Samsung 860 Evo 2TB | Samsung Evo 1TB | Samsung 970 Pro 1TB |
```
Comment (danpaul000) [inline book/src/testnet-participation.md]: Why 2TB OS drive for low end but 1TB for the other two?

Comment (sakridge) [reply book/src/testnet-participation.md]: The OS drive in the low end has to store OS + ledger/accounts, but the other ones only need to store the OS and whatever programs the machine needs.

Comment (aeyakovenko) [reply book/src/testnet-participation.md]: there are 2TB 970's now.  we should push fo 4TB of state space

Comment (aeyakovenko) [reply book/src/testnet-participation.md]: it might be doable to put 3x 2080ti and 3x970 2tb.  we need to test it out

Comment (sakridge) [reply book/src/testnet-participation.md]: https://www.tweaktown.com/news/65736/samsung-2tb-970-pro-scheduled-release/index.html

Comment (sakridge) [reply book/src/testnet-participation.md]: No 2TB 970 pros afaict

Comment (aeyakovenko) [reply book/src/testnet-participation.md]: Evo

Comment (sakridge) [reply book/src/testnet-participation.md]: 970 Pro has a 2x speed and write endurance per GB advantage over Evo. Do we need to give that up? Evo generally uses cheap nand that goes bad faster and can't withstand heavy usage so hard to recommend for a server. Although the warranty is the same. Seems that 970 evo does beat 860 pro generally, so maybe we can go with that for the OS drive.

**Pull request merged**